include/deal.II/base/work_stream.h

```diff
@@ -747,36 +747,59 @@ namespace WorkStream
 
         tf::Task last_copier;
 
+        struct CopyChunk
```

Comment:
leave a comment what this struct is (a collection of N CopyData objects, which is the output of a single worker that is chunked)